packages/lit-dev-tools-cjs/src/search/plugin.ts

```diff
@@ -59,7 +64,18 @@ export async function createSearchIndex(outputDir: '_dev' | '_site') {
   idOffset = Number(articles[articles.length - 1].objectID);
   const api: UserFacingPageData[] = await indexApi(outputDir, idOffset);
 
-  const searchIndex: UserFacingPageData[] = [...docs, ...articles, ...api];
+  idOffset = Number(api[api.length - 1].objectID);
+  const tutorials: UserFacingPageData[] = await indexTutorials(
+    outputDir,
+    idOffset
+  );
+
+  const searchIndex: UserFacingPageData[] = [
+    ...docs,
+    ...articles,
+    ...api,
+    ...tutorials,
+  ];
 
   fs.writeFileSync(OUT_PATH, JSON.stringify(searchIndex));
```

Comment:
Nit - out of scope of this change. It may be valuable to add a golden test or have a way to minimally test each of these plugins and that the resulting Algolia index looks a certain way.